concrete/elements/files/edit/image_editor/toast.php

```diff
@@ -39,6 +39,7 @@
 </div>
 
 <script>
+$(document).ready(function() {
```

Comment:
This also prevents the code from creating a global javascript variable (`imageEditor`).